projects/robots/robotis/darwin-op/worlds/darwin-op-hinge-vs-hinge2.wbt

```diff
@@ -0,0 +1,38 @@
+#VRML_SIM R2021b utf8
```

Comment:
I would rename this file `darwin-op_hinge_vs_hinge2.wbt` to be consistent with other worlds naming.